apps/story/stories/Button.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 
-import { Button } from './Button';
+import { Button } from 'ui';
```

Comment:
Can be good to rename each packages and apps name with @nextwork prefix no ?

Reply:
Yes, I think it's a good practice for the project in question. But not in the scope here for this template.